### PR TITLE
 sci-electronics/kicad: Bump to version 4.0.7 

### PR DESCRIPTION
**Pull request by zpuskas**

### CODE REVIEW

File: app-doc/kicad-doc/kicad-doc-4.0.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+DESCRIPTION="Electronic Schematic and PCB design tools manuals"
+HOMEPAGE="http://www.kicad-pcb.org/"
+SRC_URI="https://github.com/KiCad/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="|| ( GPL-3+ CC-BY-3.0 ) GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="html +pdf"
+LANGS="en fr it ja nl pl"
+for lang in ${LANGS}; do
```
Comment (jonasstein) [inline app-doc/kicad-doc/kicad-doc-4.0.7.ebuild]: local lang

Comment (zpuskas) [reply app-doc/kicad-doc/kicad-doc-4.0.7.ebuild]: I unset it immediately, but couldn't hurt to set to local.

**Comment by jonasstein** [comment]
The lib is very large (+400 MB) it needs to be downloaded if it is not cached and kicad needs to be recompiled for some reason. Is kicad usable without this large lib, if one uses local libs?
Besides that:

```
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/sci-electronics/kicad-4.0.7/work/kicad-4.0.7 ...
 * cmake-utils_src_prepare has not been run, please open a bug on https://bugs.gentoo.org/
>>> Working in BUILD_DIR: "/var/tmp/portage/sci-electronics/kicad-4.0.7/work/kicad-4.0.7_build"

```

**Comment by zpuskas** [comment]
@jonasstein There is a "miminal" USE flag where you can achieve not installing the libraries. I can take a look if the libraries can be installed separately, but that will be a non-trivial amount of work. I can make it into kicad-4.0.7-r1 or later for 4.0.8.

**Comment by jonasstein** [comment]
@zpuskas OK, I am fine with your suggestion to get a 4.0.7 in the tree first and to split it later. 
I have discussed this also with other developers, because there is no strict rule, when to split. We think it would be the best. I also made a research on https://repology.org/metapackages/?search=kicad where we see that now many others split too. This will be the best for the user and we keep the ebuild a bit more tidy and easier to maintain. 
Could you already have a look at the `cmake-utils_src_prepare` thing?

**Comment by zpuskas** [comment]
cmake-utils_src_prepare fixed. I've also removed the line ending hack as it's not needed any more.

**Comment by jonasstein** [comment]
merged sci-electronics/kicad/kicad-4.0.7.ebuild in commit a9a271d2aab2102388bdca6a65fbb5763124194c
app-doc/kicad-doc-4.0.7.ebuild does not install properly here. I suggest we get app-doc/kicad-doc-4.0.6.ebuild in the tree first and then app-doc/kicad-doc-4.0.7.ebuild You said, they are the same.

**Pull request closed without merging**